src/lib.rs

```diff
@@ -5,8 +5,12 @@ pub const KEY_RANGE: [&str; 12] = [
     "C", "C#/Db", "D", "D#/Eb", "E", "F", "F#/Gb", "G", "G#/Ab", "A", "A#/Bb", "B",
 ];
 
-const NOTE_ON_STATUS: u8 = 0b10010000;
-const NOTE_OFF_STATUS: u8 = 0b10000000;
+const NOTE_ON_STATUS: u8 = 0b1001_0000;
+const NOTE_OFF_STATUS: u8 = 0b1000_0000;
+
+const REFRESH_PORTS_ERROR_MSG: &str = "Couldn't refresh input ports";
```

Comment:
In `MidiOutput`'s `refresh_ports` method, it'd be _output ports_ and not _input ports_.